package mx.edu.utez.proyecto1D.controller.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class RentaResponseDTO {
    private String nombreCliente;
    private double costoTotal;
    private String mensaje;
}